### PR TITLE
Reactredux

### PR DESCRIPTION
**Pull request by businesskaniki**
- [ ] integrated react with  redux

### CODE REVIEW

File: src/components/Nav.jsx [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
         <h1>Bookstore CMS</h1>
         <nav className="navigation">
-          <Link to="/" className="link">books</Link>
+          <Link to="Ui" className="link">books</Link>
           <Link to="/Categories" className="link">Categories</Link>
         </nav>
       </header>
       <Routes>
-        <Route path="/" element={<Books />} />
+        <Route path="Ui" element={<Ui />} />
```
Comment (TedLivist) [inline src/components/Nav.jsx]: - [ ] When I open your app, nothing is displayed. You need to change this "Ui" to the root route, like so:

  ```
  <Route path="/" element={<Ui />} />
  ```

File: src/components/Nav.jsx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+import Ui from './Ui';
 
 function Nav() {
   return (
     <>
       <header className="head">
         <h1>Bookstore CMS</h1>
         <nav className="navigation">
-          <Link to="/" className="link">books</Link>
+          <Link to="Ui" className="link">books</Link>
```
Comment (TedLivist) [inline src/components/Nav.jsx]: - [ ] This nav link should also be changed to "/" as it was before

File: src/components/Nav.jsx [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
           <Link to="/Categories" className="link">Categories</Link>
         </nav>
       </header>
       <Routes>
-        <Route path="/" element={<Books />} />
+        <Route path="Ui" element={<Ui />} />
         <Route path="/Categories" element={<Categories />} />
       </Routes>
```
Comment (TedLivist) [inline src/components/Nav.jsx]: - [ ] In keeping with convention, routing should not be defined in the Navbar component. Instead, it should be in the App component. Kindly move this code from here to the App component. Under the `<Nav />` link

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @businesskaniki,

Good job so far!
✔️ Books can be added and removed
✔️ Books are removed using unique ID
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @businesskaniki, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Linters ✔
- Components ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**